lib/RSS.js

```diff
@@ -11,13 +11,17 @@ module.exports = class RSS extends Helper {
 
   _formatPage(res, date) {
     const data = res.rss.channel.item;
-
-    return {
+    const result = {
       response_time: parseInt(date, 10),
       page: NaN,
-      total_results: data.length,
+      total_results: 0,
       total_pages: NaN,
-      results: data.map(item => {
+      results: []
+    };
+
+    if(data !== undefined) {
+      data['total_results'] = data.length;
```

Comment:
Should refer to the `result` object not the `data` object. Also please use the dot notation, instead of the square bracket notation for accessing the `total_results` attribute.